x/gov/keeper/keeper.go

```diff
@@ -230,3 +234,12 @@ func (k Keeper) assertSummaryLength(summary string) error {
 	}
 	return nil
 }
+
+// assertVoteOptionsLen returns an error if given vote options length
+// is greater than a pre-defined MaxVoteOptionsLen.
+func (k Keeper) assertVoteOptionsLen(options v1.WeightedVoteOptions) error {
+	if uint64(len(options)) > k.config.MaxVoteOptionsLen {
```

Comment:
We need to only check this is the MaxVoteOptions > 0

Reply:
Right, addressed